analyzer/src/test/kotlin/AnalyzerResultBuilderTest.kt

```diff
@@ -17,17 +17,25 @@
  * License-Filename: LICENSE
  */
 
-package org.ossreviewtoolkit.model
+package org.ossreviewtoolkit.analyzer
 
 import com.fasterxml.jackson.module.kotlin.readValue
 
 import io.kotest.core.spec.style.WordSpec
 import io.kotest.matchers.should
 import io.kotest.matchers.shouldBe
 
+import org.ossreviewtoolkit.model.AnalyzerResult
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.OrtIssue
+import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.Project
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.Scope
+import org.ossreviewtoolkit.model.yamlMapper
 import org.ossreviewtoolkit.utils.test.containExactly
 
-class AnalyzerResultTest : WordSpec() {
+class AnalyzerResultBuilderTest : WordSpec() {
```

Comment:
How about adding a test which tests that a curation from a provider gets applied?

Reply:
I'd say that's out of scope of this refactoring, and I'd leave it for the future.